### PR TITLE
Config file populated with site-specific data paths examples

### PR DESCRIPTION
**Pull request by valeriupredoi**
Before you start, please read [CONTRIBUTING.md](https://github.com/ESMValGroup/ESMValCore/blob/master/CONTRIBUTING.md).

**Tasks**

-   [x] [Create an issue](https://github.com/ESMValGroup/ESMValCore/issues) to discuss what you are going to do, if you haven't done so already (and add the link at the bottom)
-   [x] This pull request has a descriptive title that can be used in a changelog

* * *

Closes https://github.com/ESMValGroup/ESMValTool/issues/1622

### CODE REVIEW

**Comment by valeriupredoi** [comment]
okey dokey guys, we need more rootpaths/drs's but beats me which are which apart from my beloved Jasmine :beer:

**Comment by valeriupredoi** [comment]
should add this https://github.com/ESMValGroup/ESMValCore/issues/582 here too - also, we need ETHZ, maybe @bascrezee can add it 'ere? :beer:

**Comment by valeriupredoi** [comment]
yeah god call @bouweandela - I have just added the paths for Jasmin and restructured the file, @mattiarighi can you do the same pls mate :beer:

**Comment by bascrezee** [comment]
I'll add the ETHZ paths now.

File: esmvalcore/config-user.yml [hunk 1]
```diff
@@ -39,16 +39,62 @@ config_developer_file: null
...
+#  OBS6: /group_workspaces/jasmin4/esmeval/obsdata-v2
+#  obs4mips: /group_workspaces/jasmin4/esmeval/obsdata-v2
+#  ana4mips: /group_workspaces/jasmin4/esmeval/obsdata-v2
+#  CORDEX: /badc/cordex/data/CORDEX/output
+#drs:
+#  CMIP6: BADC
+#  CMIP5: BADC
+#  CMIP3: BADC
+#  CORDEX: BADC
+#  OBS: BADC
```
Comment (bouweandela) [inline esmvalcore/config-user.yml:74]:
```suggestion
#  OBS: BADC
#  OBS6: BADC
```

Comment (bouweandela) [reply esmvalcore/config-user.yml:74]: OBS6 was still missing

**Comment by bouweandela** [comment]
@mattiarighi Would you be able to add the missing projects to the DKRZ configuration? Then we can get this merged.

**Comment by mattiarighi** [comment]
Done.

**Pull request merged**